### DISCUSSION TITLE
Can data payloads stored in the same collection have different fields?

### DISCUSSION THREAD

**Discussion by Friedrich-hue** [qdrant/qdrant#2792] [Q&A]
Can data payloads stored in the same collection have different fields?

For example:
{
source: 'test'
}

{
page: 1
}

**Comment by timvisee** [comment]
Yes! There is no fixed structure for payload. In other words: every point in your collection could have a different payload structure.

However, if you want to index payload fields. You likely want to use a consistent structure for indexing to work.
